module/solutions/helper.js

```diff
@@ -291,9 +300,14 @@ module.exports = class SolutionsHelper {
 	 * @returns {JSON} solution creation data.
 	 */
 
-	static createSolution(userToken, solutionData, checkDate = false) {
+	static createSolution(solutionData, checkDate = false) {
 		return new Promise(async (resolve, reject) => {
 			try {
+				if (!solutionData.scope || !Object.keys(solutionData.scope).length > 0) {
```

Comment:
@Prajwal17Tunerlabs see if we can incorporate this to validation

Reply:
scope validation fixed